### DISCUSSION TITLE
Welcome to dspira-lessons Discussions!

### DISCUSSION THREAD

**Discussion by TomMHO** [WVURAIL/dspira-lessons#15] [General]
### Discussed in https://github.com/WVURAIL/dspira-lessons/discussions/6

<div type='discussions-op-text'>

<sup>Originally posted by **PranavSanghavi** December 17, 2020</sup>

## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

</div>

**Comment by TomMHO** [comment]
Hello Everybody,

I really appreciate the outstanding efforts of DSPIRA in making this project available to hobbyists like me.  I've built both the small and large horns and they both work very well.

However, I did have a problem when I downloaded the ubuntu_radio_astro2021.zip  iso image and flashed it to a 32 GB USB stick.  Ubuntu tried to start, but the desktop never opened.  I tried it again with a 64 GB stick and it works fine.  Anybody else notice this?

Thanks again,
Tom Hagen
Rochester MI

**Comment by kbandura** [comment]
All USB sticks are not created equal.  You may have luck just flashing the 32 again as well.

**Comment by TomMHO** [comment]
The 32G is an old stick, and I just purchased the 64G, so that could be the problem.  And I did try flashing the 32G stick twice to no avail.  :-)

Thanks,
Tom

**Comment by glangsto** [comment]
Hi

There are some troubles with using devices bigger than 32GB.  I’m not sure
if that is the use or not, though.

Alternatively this link and installation guide have been successfully used
by several teachers.

https://drive.google.com/file/d/1ZimSdkMnhlHTJFUAVnjuZ4X0ezROjXwY/view?usp=sharing

The installation guide is here:

https://docs.google.com/document/d/1PfLD4Qs_Lqaym5gc8UixnePTN2Mfi6LW_h2DaOnnrT0/edit?usp=sharing

Glen

Glen I Langston, Ph. D.
Galactic Astronomy Program Director
National Science Foundation
304-456-3032

**Comment by TomMHO** [comment]
Thanks all, for the help.

I did a little more investigation and I found out that the original 32 GB USB stick I used has very slow read/write speeds of 5/5 MB/s.  I purchased a new Samsung 32 GB stick (advertised at 200 MB/s read speed) and the read/write speed was measured at 211/26 MB/s.  I flashed the spectrometer image onto this drive and it runs great!  (I'm using a USB speed check utility I found online to check speeds.)

So it looks to me like the USB R/W speed is the important factor and I will use the fastest stick I can get to run the spectrometer.

Tom Hagen

**Comment by rbaker314** [comment]
Hi Tom,

What is the name of the USB  speed check utility?

Thanks,

Robert Baker

**Comment by TomMHO** [comment]
Hi Robert:  It's USB Flash Benchmark,  https://www.majorgeeks.com/files/details/usb_flash_benchmark.html

I'm always leery about downloading these "no-name" executables, but I don't think I got any viruses or trojans from this one!

I saw some online tests too, maybe you could try one and let us know how it went?

Tom

**Comment by glangsto** [comment]
Glen I Langston, Ph. D.
Galactic Astronomy Program Director
National Science Foundation
304-456-3032
